packages/webview_flutter/webview_flutter_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.9.1
```

Comment:
Based on the discussion in the other issue you could remove the version change and make this `NEXT` since I was wrong about it affecting clients. But it's fine to leave the version change as well.

Reply:
Let's leave this version.